### ISSUE TITLE
Setting tax exempt in wrong map

### ISSUE THREAD

**Issue by joze144** [carlsbad-sunshine/lid-contracts#1]
This line think is incorrect, you're setting taxExempt in the wrong line

https://github.com/carlsbad-sunshine/lid-contracts/blob/ef1fb851f9c33c9c9a8d3bd8d47977ee87a25624/contracts/LidToken.sol#L85

Was going through your contracts which seem quite cool and found this potential issue, not sure if it's even used though.

**Comment by carlsbad-sunshine** [comment]
Hmm interesting, Ill check on this.

**Comment by carlsbad-sunshine** [comment]
@joze144 Good catch, you're correct. Will update once a fix is done, double checking theres no unexpected effects.

**Comment by Onixmiha** [comment]
ef1fb851f9c33c9c9a8d3bd8d47977ee87a25624
